dockers/common/dynamodb-local/release.sh

```diff
@@ -1,17 +1,17 @@
 set -ex
 # SET THE FOLLOWING VARIABLES
 # docker hub username
-USERNAME=nhsdev
+USRNAME=nhsdev
```

Comment:
Is that renaming needed?

Reply:
It was taken my username from system veriables (alicjab) when it was USERNAME, not the one specified in the file. Now it always taken the one from file.

Reply:
USERNAME=nhsdev would override anything in your system environment. I'd revert this it should work as previously written.

Reply:
reverted